content/security.md

```diff
@@ -5,48 +5,53 @@ We use [`OAuth2`](https://oauth.net/2/) to authenticate users in the API and
 
 ## Authentication
 
-### Create an OAuth client
+### Create an API connection
 
 The first operation to do is to authorize third-party applications to use the API. For that, you need to create a client id.
 A client id identifies a client application.
 
-To increase security, the OAuth protocol specifies that a client id comes with a secret. They are always generated, used and revoked together.
+To increase security, the OAuth protocol specifies that a client id comes with a secret. They are always generated, used and revoked together. This set of credentials is called `API connection` inside the PIM.
 
-To create a new pair of client id / secret, use the following command:
+There are two ways to create these credentials.
 
+#### Via the PIM UI (2.0 only)
```

Comment:
`Through` ? `Via` is not a recommend word to use in french, and I think it's the same thing in english

Reply:
Will change